src/content/overview/fleet-management/cluster-management/installation-name/index.md

```diff
@@ -0,0 +1,44 @@
+---
+linkTitle: Installation names
+title: Picking an installation name
+description: Every Giant Swarm installation has a unique name. Learn the rules to select a proper name for a new installation.
+weight: 100
+menu:
+  principal:
+    parent: overview-fleet-management-clusters
+    identifier: overview-fleet-management-clusters-installation-name
+user_questions:
+  - How do I determine the name for my Giant Swarm installation / management cluster?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
```

Comment:
I think it's normally KaaS teams creating installations, so I'd say phoenix/rocket show own this.